codegen/codegen.go

```diff
@@ -463,6 +452,12 @@ func (cg *CodeGen) EmitCallStmt(ctx context.Context, scope *parser.Scope, call *
 		}
 	}
 
+	// Yield before executing the next call statement.
+	err = cg.Debug(ctx, scope, call.Name, ret, opts)
```

Comment:
I moved this down here from above so `opts` would be available. I'm not 100% sure that's safe, and would appreciate confirmation.

Reply:
I believe this is necessary but I want to point out that this changes the behavior from stepping in this order: call -> args, to args -> call. Since args need to be evaluated and passed by value, I think this change makes sense.

You should try out stepping through the debugger to see the difference.

Reply:
Ah yes, I did notice that, but didn't realize it was a result of this change.